tsconfig.json

```diff
@@ -18,7 +18,8 @@
       "@material-ui/lab": ["./material-ui-lab/src"],
       "@material-ui/lab/*": ["./material-ui-lab/src/*"],
       "@material-ui/system": ["./material-ui-system/src"]
-    }
+    },
+    "types": []
```

Comment:
Do we need this change?

Reply:
Yes, otherwise we get definition conflicts with all `@types` packages that requires `@types/react-native`, in this case, it was `@types/styled-components` that caused issues